### DISCUSSION TITLE
Tutorial / HowTo about setup a Python project using GNU gettext

### DISCUSSION THREAD

**Discussion by buhtz** [WeblateOrg/weblate#8526] [Q&A] [answered]
Is there a tutorial or howto about how to setup a weblate translation/component for a Python project (on a Gitea host) using GNU gettext (po-files)?

The Weblate documentation doesn't help here because it is IMHO more a reference then a manual.

Related: https://github.com/WeblateOrg/weblate/discussions/8532

**Comment by nijel** [comment, marked as answer]
There is generic gettext tutorial - https://docs.weblate.org/en/latest/devel/gettext.html

It will work similarly in Python, you can use the same tools there and the gettext API is same as well.

**Comment by buhtz** [comment]
There is a discussion on Codeberg.org about how to setup such a project. Codeberg.org does host its own weblate instance for their users.
https://codeberg.org/Codeberg/Community/issues/965
